### PR TITLE
🚀 Release v1.9.0-beta.1

### PR DESCRIPTION
**Pull request by chandankumar4**
**What this PR does / why we need it**:

Add release note for v1.9.0-beta.1

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Part Of: https://github.com/kubernetes-sigs/cluster-api/issues/11092

/area release

### CODE REVIEW

**Comment by sbueringer** [comment]
Overall looks good so far. Please:
* Manually drop the broken link: https://github.com/kubernetes-sigs/cluster-api/actions/runs/11908161174/job/33183077594?pr=11441
* Re-generate the release notes slightly before the release to get the latest merged PRs


/approve

**Review by Sunnatillo — COMMENTED** [review]
/hold
/lgtm

File: CHANGELOG/v1.9.0-beta.1.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
+🚨 This is a BETA RELEASE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.28.x -> v1.31.x
+- Workload Cluster: v1.26.x -> v1.31.x
+
+## Deprecation and Removals Warning
```
Comment (sbueringer) [inline CHANGELOG/v1.9.0-beta.1.md:8]: Just finished a draft for highlights and deprecation and removals. If possible please replace this section with the highlights and deprecation sections from here: https://github.com/kubernetes-sigs/cluster-api/issues/11092#issuecomment-2402397477

**Comment by sbueringer** [comment]
Thx! Especially for picking up last minute changes :)

/lgtm

**Pull request merged**